CONTRIBUTING.md

```diff
@@ -84,9 +82,11 @@ Before you submit your pull request consider the following guidelines:
      git checkout -b my-fix-branch master
      ```
 
+* Setup your editor http://editorconfig.org/#download
```

Comment:
maybe we can do:
`[Setup your editor](http://editorconfig.org/#download)`

instead of just having the link